### PR TITLE
Git branch strategies

### PR DESCRIPTION
**Pull request by calebwoods**
Refactoring of existing tools

### CODE REVIEW

File: git/merge-strategy.md [hunk 1]
```diff
@@ -39,33 +39,11 @@ git checkout master
...
-```
-
-#### Abbreviations
-
-|Tasks            |Source Control
-|:----------------|:-------------
-|  KB - Kanbanery |  GH - Github
-|  TR - Trello    |  GL - Gitlab
-|  LK - Lean Kit  |  BB - BitBucket
+When creating running the above command you will be prompted for a commit message for your merge. The commit message used the [Merge Commit Format](merge-commit-format.md).
```
Comment (timirwin) [inline git/merge-strategy.md]: Is `When creating running` correct? I'm not sure I understand what you mean.

Comment (calebwoods) [inline git/merge-strategy.md]: That is awkward.  Pushed a fix

File: git/merge-commit-format.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<highlights of feature change if any>
+```
+
+And example would look like:
+
+```
+Return 404 on permissions API when no access [TR #45] (GH #4)
+
+If a user no access to a study at all we want to be able communicate that to API clients, thus we will return a 404.
```
Comment (wesrich) [inline git/merge-commit-format.md]: Should this be `If a user cannot access`, or similar?

File: git/merge-strategy.md [hunk 3]
```diff
@@ -39,33 +39,11 @@ git checkout master
...
-|:----------------|:-------------
-|  KB - Kanbanery |  GH - Github
-|  TR - Trello    |  GL - Gitlab
-|  LK - Lean Kit  |  BB - BitBucket
+When running the above command you will be prompted for a commit message for your merge. The commit message should follow the [Merge Commit Format](merge-commit-format.md).
 
 ### Push the Merged Changes to the Remote
 
-You now have a fully merge repository, but the remote does not.  You need to push those changes now: 
+You now merge complete locally, but the remote does not.  You need to push those changes now:
```
Comment (wesrich) [inline git/merge-strategy.md]: This wording feels a little odd as well. Perhaps `Your merge is now completed locally but needs to be updated on the remote repository.` ?

File: git/merge-commit-format.md (hunk 2 above)
Comment (calebwoods) [inline git/merge-commit-format.md]: done

File: git/merge-strategy.md (hunk 3 above)
Comment (calebwoods) [inline git/merge-strategy.md]: done

File: git/squashing.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+This shows all the commits that will be replayed.  To squash everything into a single commit we just change every commit after the first to squash:
+
+```
+pick 5eb055d Change Skype Standup document to focus on Slack
+squash 4f91fe4 Fix standup link
+squash 57ae0ea Change Skype standup to Slack standup
+squash 1f80132 recommend advanced notice for extended time off
+```
+
+Save and close the file.  This then open your editor to edit the commit message:
```
Comment (wesrich) [inline git/squashing.md]: `opens` needs an `s` :)

File: git/squashing.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+# This is the 3rd commit message:
+
+Change Skype standup to Slack standup
+
+# This is the 4th commit message:
+
+recommend advanced notice for extended time off
+```
+
+Edit this to be the finial commit message according to the our standards, which may look something like:
```
Comment (timirwin) [inline git/squashing.md]: `finial`? Is that supposed to be `final`?

**Pull request merged**